spec/features/tableau_de_bord_spec.rb

```diff
@@ -4,46 +4,119 @@
 require 'support/api_ban_helper'
 
 feature "J'ai accès à mes dossiers depuis mon tableau de bord" do
-  let(:projet)      { create(:projet, :prospect, :with_invited_operateur) }
-  let(:operateur)   { projet.invited_operateur }
-  let(:instructeur) { create :instructeur, departements: [projet.departement] }
-  let(:pris)        { create :pris,        departements: [projet.departement] }
+  let(:projet)      { create(:projet, :en_cours_d_instruction, :with_invited_pris) }
+  let(:operateur)   { projet.operateur }
+  let(:instructeur) { projet.invited_instructeur }
+  let(:pris)        { projet.invited_pris }
+  let(:agent_operateur)   { create :agent, :operateur, intervenant: operateur}
+  let(:agent_instructeur) { create :agent, :instructeur, intervenant: instructeur}
+  let(:agent_pris)        { create :agent, :pris, intervenant: pris}
 
-  before { login_as agent, scope: :agent }
+  before do
+    login_as current_agent, scope: :agent
+    projet.update_attributes agent_instructeur: agent_instructeur, agent_operateur: agent_operateur
+  end
 
   context "en tant qu'opérateur" do
-    let(:agent) { create :agent, intervenant: operateur }
+    let(:current_agent) { agent_operateur }
 
-    scenario "je peux accéder à un dossier à partir du tableau de bord" do
+    scenario "j'ai accès au tableau de bord avec toutes les informations disponibles" do
       visit dossiers_path
       within "#projet_#{projet.id}" do
-        expect(page).to have_content(I18n.t("projets.statut.prospect"))
+        expect(page).to have_content(projet.numero_plateforme)
+        expect(page).to have_content(projet.opal_numero)
+        expect(page).to have_content(projet.demandeur_principal.fullname)
+        expect(page).to have_content(projet.adresse.region)
+        expect(page).to have_css('td#departement')
```

Comment:
Normalement on doit même pouvoir tester que la cellule a le bon contenu, avec quelque chose comme :

```ruby
expect(page).to have_css('td#departement', text: projet.departement)
```

https://makandracards.com/makandra/777-using-the-full-power-of-have_css